cnd/migrations/2020-05-06-105000_created-swap-tables/up.sql

```diff
@@ -0,0 +1,64 @@
+-- Your SQL goes here
+
+CREATE TABLE swaps
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    local_swap_id UNIQUE	NOT NULL,
+    role			NOT NULL,
+    counterparty_peer_id	NOT NULL
+);
+
+CREATE TABLE secret_hashes
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    secret_hash		NOT NULL,
+    FOREIGN KEY (swap_id)	REFERENCES swaps(id)
+);
+
+CREATE TABLE shared_swap_ids
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    shared_swap_id		NOT NULL,
+    FOREIGN KEY (swap_id)	REFERENCES swaps(id)
+);
+
+CREATE TABLE address_hints
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    address_hint		NOT NULL,
+    FOREIGN KEY(swap_id)	REFERENCES swaps(id)
+);
+
+CREATE TABLE herc20_swap_details
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    amount			NOT NULL,
+    chain_id			NOT NULL,
+    expiry			NOT NULL,
+    hash_function		NOT NULL,
```

Comment:
Don't think this should be configurable, would be a different protocol then.

Reply:
I thought I just responded? 🧐 Mhmm... message gone...

Nothing in particular. This property just was there somehow there. Feel free to remove it/don't add it. It does not add much value as it is always static.

Reply:
Well, you edited my comment :joy: 

![image](https://user-images.githubusercontent.com/5486389/81526133-25676080-939a-11ea-91cc-0823d2a5a9d1.png)

Thanks for clarifying :)

Reply:
haha... ups 😅